### ISSUE TITLE
Many things in Home screen are not operational through a keyboard (Tab) interface.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#811]
**Describe the bug**
Many things in Home screen are not operational through keyboard standard keys. If we want to access different buttons on the page such as "Add New", "Search" etc, then we are not able to select  them through keyboard.

**To Reproduce**
1. Go to Home screen(http://localhost:9992/cm/app/?view=home&pathType=unknown)
2. See that all the options/controls on the screen are not accessible using standard Keyboard keys like Tab key.

**Expected behavior**
Whole page should be accessible using Standard keyboard Keys.
